kolibri/core/assets/src/views/KTooltip.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+
+  <popper
+    v-if="readyToInit"
+    :reference="htmlElement"
+    :disabled="disabled"
+    :visibleArrow="false"
+    :options="options"
+    trigger="hover"
+    class="k-tooltip-container"
+  >
+    <div
+      dir="auto"
+      class="k-tooltip"
+    >
+      <!--Default slot that will contain the tooltip content.-->
+      <slot></slot>
+    </div>
+  </popper>
+
+</template>
+
+
+<script>
+
+  import Popper from 'vue-popperjs';
+  import isArray from 'lodash/isArray';
+
+  /**
+   * Used to create a tooltip.
+   */
+  export default {
+    name: 'KTooltip',
+    components: {
+      Popper,
+    },
+    props: {
+      /**
+       * String of ref which tooltip will be positioned relative to.
+       */
+      reference: {
+        type: String,
+        required: true,
+      },
+      /**
+       * Refs object.
```

Comment:
Maybe write more here explaining why you need to pass parents' refs into KTooltip